tests/jest/config/webpack/loaders/pretendPackageThree.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "terra-dev-site",
+  "version": "6.2.0",
+  "repository": {
```

Comment:
Can you add a description key to these fake packages to describe what it's intent is?

Reply:
Added here : https://github.com/cerner/terra-dev-site/pull/243/commits/3acf9f958a72e6ad031400f2c566813e15a3c78d